Hcal/src/Hcal/HcalSimpleDigiAndRecProducer.cxx

```diff
@@ -0,0 +1,149 @@
+#include "Hcal/HcalSimpleDigiAndRecProducer.h"
+
+namespace hcal {
+
+void HcalSimpleDigiAndRecProducer::configure(
+    framework::config::Parameters& ps) {
+  input_coll_name = ps.getParameter<std::string>("input_coll_name");
+  input_pass_name = ps.getParameter<std::string>("input_pass_name");
+  output_coll_name = ps.getParameter<std::string>("output_coll_name");
+  mev_per_mip = ps.getParameter<double>("mev_per_mip");
+  pe_per_mip = ps.getParameter<double>("pe_per_mip");
+  attenuation_length = ps.getParameter<double>("attenuation_length");
+  readout_threshold = ps.getParameter<int>("readout_threshold");
+  mean_noise = ps.getParameter<double>("mean_noise");
+  position_resolution = ps.getParameter<double>("position_resolution");
```

Comment:
also if we follow our (not-yet-written-down) coding rules, these should end with a trailing underscore, showing that these are global variables

Reply:
will do :+1: